src/styles/style.css

```diff
@@ -0,0 +1,2 @@
+@import 'blocks/stars.css';
+@import 'blocks/resets.css';
```

Comment:
It's generally a good practice to import reset styles before any other styles so that all other styles can override the default styles as needed. Consider changing the order of imports so that 'blocks/resets.css' comes first.